outline/data_structures.md

```diff
@@ -289,24 +289,26 @@ be confusing.
 > After the creation, we want to save a new value associated to the
 > key. The `assoc` function can be used by assigning a new value to
 > the existing key.
-> Also, there's handy function `update-in`. The function takes map and
-> a key or list of keys with a function. The value of specified key will be the
-> first argument of the given function.
+> Also, there's handy function `update`. The function takes map and
+> a key with a function. The value of specified key will be the first
+> argument of the given function.
```

Comment:
This could be clearer. How about something like this?

> Also, there's a handy `update` function which takes three arguments:
>
>   - a map;
>   - a key; and
>   - a function to transform the value of the specified key in the given map.

This doesn't mention the option of providing additional arguments to the transformation function up front, but doing so might make the explanation confusing.